extensions/adobe/experience/marketo/marketo-lead.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/marketo-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Lead",
+  "type": "object",
+  "description": "Marketo Lead schema. Includes other Profile mixins like person/work details, opt in/out etc.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-lead-crm",
+    "https://ns.adobe.com/experience/marketo/marketo-contact-lead-person"
+  ],
+  "definitions": {
+    "marketo-lead": {
+      "properties": {
+        "marketo:leadID": {
+          "title": "Marketo Lead ID",
+          "description": "Marketo lead id.",
+          "type": "string"
+        },
+        "marketo:instanceID": {
+          "title": "Instance Id",
+          "description": "Source System Id.",
+          "type": "string"
+        },
+        "marketo:personID": {
+          "title": "Marketo Person Id",
+          "description": "Marketo person id.",
+          "type": "string"
+        },
+        "marketo:contactID": {
+          "title": "Marketo Contact Id",
+          "description": "Marketo contact id.",
+          "type": "string"
+        },
+        "marketo:accountID": {
+          "title": "Marketo Account Id",
+          "description": "Marketo account id.",
+          "type": "string"
+        },
+        "marketo:abmProposedNameID": {
+          "title": "Account Based Marketing Proposed Name Id",
+          "description": "Account based marketing proposed name id.",
+          "type": "string"
+        },
+        "marketo:count": {
+          "title": "Count",
+          "description": "Count.",
+          "type": "number"
+        },
+        "marketo:totalProposedNameCount": {
+          "title": "Total Proposed Name Count",
+          "description": "Total proposed name count.",
+          "type": "number"
+        },
+        "marketo:leadAddress": {
```

Comment:
How different is this from xdm: address in the profile-work-address/profile-personal-details